bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -372,6 +377,17 @@ createFlowFunction(const BinaryFunction::BasicBlockOrderType &BlockOrder) {
     }
   }
 
+  // Add dummy edges from the exit blocks to the sink block.
+  for (uint64_t I = 1; I < BlockOrder.size() + 1; I++) {
+    FlowBlock &Block = Func.Blocks[I];
+    if (Block.IsExit) {
```

Comment:
Nit: use "early continue" (https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code). Since `Block` is not used anywhere else, I'd just write:
```
  if (!Func.Blocks[I].IsExit)
    continue;